admin_manual/ai/app_assistant.rst

```diff
@@ -160,3 +160,17 @@ This field is appended to the block of chat messages, i.e. attached after the me
 
 The number of latest messages to consider for generating the next message. This does not include the user instructions, which is always considered in addition to this. This value should be adjusted in case you are hitting the token limit in your conversations too often.
 The AI text generation provider should ideally handle the max token limit case.
+
+Improve AI processing throughput
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Most AI tasks will be run as part of the background job system in Nextcloud which only runs jobs every 5 minutes by default.
+To pick up scheduled jobs faster you can set up background job workers that process AI tasks as soon as they are scheduled:
+
+run the following occ commands a daemon (you can also spawn multiple, for parallel processing):
+
+.. code-block::
+
+   occ background-job:worker 'OC\TaskProcessing\SynchronousBackgroundJob'
+
+Make sure to restart these daemons regularly. For example once a day.
```

Comment:
Let's maybe explain why. Something like:
"Apps and server update won't be effective in running daemons, that's why they need to be restarted from time to time."